sound/soc/sof/sof-priv.h

```diff
@@ -162,6 +162,9 @@ struct snd_sof_dsp_ops {
 	int (*send_msg)(struct snd_sof_dev *sof_dev,
 			struct snd_sof_ipc_msg *msg); /* mandatory */
 
+	/* sdw */
+	bool (*check_sdw_irq)(struct snd_sof_dev *sdev); /* optional */
```

Comment:
I still feel that checking sdw irq is not very common in sof ops

Reply:
> I still feel that checking sdw irq is not very common in sof ops

How about adding check_sdw_irq ops into [struct sof_intel_dsp_desc](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/intel/shim.h#L155)? I will follow the majority. @plbossart @RanderWang @ranj063

Reply:
using struct sof_intel_dsp_desc sounds good to me, you already have SSP information in there.

Reply:
sure, a good idea